benchmarks/bin/php_no_xdebug

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+temporaryPath="$(mktemp -t php.XXXX).ini"
+
+# Using awk to ensure that files ending without newlines do not lead to configuration error
+php -i | grep "\.ini" | grep -o -e '\(/[a-z0-9._-]\+\)\+\.ini' | grep -v xdebug | xargs awk 'FNR==1{print ""}1' > "$temporaryPath"
```

Comment:
There's no need for that on Travis at least: https://github.com/amphp/mysql/blob/67f101ed261407111419dc2ed10ab1e294d722ea/.travis.yml#L57

Reply:
As you suggested, now I remove all configuration with `-n`, so think that there no need for this configuration anymore.